### ISSUE TITLE
Can't call service thermiq_mqtt.write_reg

### ISSUE THREAD

**Issue by barvidss** [ThermIQ/thermiq_mqtt-ha#16]
## Version of the custom_component

ThermIQ-MQTT v1.2.2

## Configuration

```thermiq_mqtt:
  mqtt_node: ThermIQ/ThermIQ-mqtt-bb
  #thermiq_dbg: False

```

## Describe the bug
When I try to call the service thermiq_mqtt.write_reg with the example data I get an unknown error in the UI and "AttributeError: 'int' object has no attribute 'lower'" in the Core log.

## Debug log

```text

2021-12-23 23:18:03 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548352004928] Error handling message: Unknown error
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 27, in _handle_async_response
    await func(hass, connection, msg)
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 527, in handle_execute_script
    await script_obj.async_run(msg.get("variables"), context=context)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1260, in async_run
    await asyncio.shield(run.async_run())
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 363, in async_run
    await self._async_step(log_exceptions=False)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 381, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 584, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1495, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1532, in _execute_service
    handler.job.target(service_call)
  File "/config/custom_components/thermiq_mqtt/__init__.py", line 372, in write_reg_service
    reg = reg.lower()
AttributeError: 'int' object has no attribute 'lower'

**Comment by ThermIQ** [comment]
Description of service is misleading, should be:
      description: Register number, either in hex as "rXX" or in decimal "dNN" to write to
      example: 'r32 or  d50'

Documentation will be corrected

**Comment by ThermIQ** [comment]
Closed in v1.2.3

**Comment by ThermIQ** [comment]
You could also use the service "write_id" which uses the functional name instead of register name to make your automations more easy to understand
